.expeditor/bundle_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -evx
+
+gem environment
+bundler_version=$(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
```

Comment:
this should probably be renamed because bundler uses `ENV['BUNDLER_VERSON']` now to setup trampolining, but that code is buggy and we don't want to use it, at at a minimum this is going to cause confusion to humans later.